infra/cifuzz/clusterfuzz_deployment.py

```diff
@@ -133,23 +132,23 @@ def download_corpus(self, target_name):
     return corpus_dir
 
   def _get_build_name(self):
-    return self.BASE_BUILD_NAME + self.config.sanitizer
+    return self.config.sanitizer + '-latest'
 
   def _get_corpus_name(self, target_name):  # pylint: disable=no-self-use
     """Returns the name of the corpus artifact."""
-    return 'corpus-{target_name}'.format(target_name=target_name)
+    return target_name
 
   def _get_crashes_artifact_name(self):  # pylint: disable=no-self-use
     """Returns the name of the crashes artifact."""
-    return 'crashes'
+    return 'current'
```

Comment:
I don't really understand why things like `latest` or `current` help in artifact names.